iroh/src/node/builder.rs

```diff
@@ -454,7 +455,10 @@ where
 
     async fn build_inner(self) -> Result<ProtocolBuilder<D>> {
         trace!("building node");
-        let lp = LocalPoolHandle::new(num_cpus::get());
+        let lp = LocalPool::new(local_pool::Config {
+            panic_mode: PanicMode::LogAndContinue,
```

Comment:
Womp womp, this is because of https://github.com/tokio-rs/tracing/issues/2870, mostly